student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   def initialize(classroom, age, name = 'unknown', parent_permmission: true)
     super(age, name, parent_permmission)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)

```

Comment:
- [ ] **Required:** Kindly update this class to include the classroom setter and an accessor to get it. Consider the code below:

```ruby
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```